infra-gen2/backends/storage/main/amplify/backend.ts

```diff
@@ -0,0 +1,38 @@
+import { defineBackend } from "@aws-amplify/backend";
+import * as s3 from "aws-cdk-lib/aws-s3";
+import { auth } from "./auth/resource";
+import { storage } from "./storage/resource";
+
+/**
+ * @see https://docs.amplify.aws/react/build-a-backend/ to add storage, functions, and more
+ */
+const backend = defineBackend({
+  auth,
+  storage,
+});
+
+// custom storage configurations
+const s3Bucket = backend.storage.resources.bucket;
+const cfnBucket = s3Bucket.node.defaultChild as s3.CfnBucket;
+
+cfnBucket.accelerateConfiguration = {
+  accelerationStatus: "Enabled",
+};
+
+backend.storage.resources.cfnResources.cfnBucket.corsConfiguration = {
```

Comment:
The same comment should go here

```suggestion
// required to add the metadata header, which amplify-backend does not support
backend.storage.resources.cfnResources.cfnBucket.corsConfiguration = {
```